docs/software/system_stats.md

```diff
@@ -93,6 +94,27 @@ DietPi-Dashboard is a very lightweight and standalone web interface for monitori
     journalctl -u dietpi-dashboard
     ```
 
+=== "Password Protection"
+
+    Password protection is enabled by default from DietPi 7.9 and on, but if you installed it before, you'll need to make these changes in the config file:
+    ```toml
+    pass = true
+    # You can get the hash by running "echo -n '<PASSWORD>' | sha512sum
+    hash = "SHA512 hash of password"
+    # You can get a random secret by running "openssl rand -hex 32"
+    secret = "Random 64-character secret"
+    ```
```

Comment:
```suggestion


    ```toml
    pass = true
    # You can get the hash by running "echo -n '<PASSWORD>' | sha512sum"
    hash = "<SHA512 hash of PASSWORD>"
    # You can get a random secret by running "openssl rand -hex 32"
    secret = "<random 64-character secret>"
    ```
```
Btw, are there restrictions for characters of the secret? Hexadecimal characters are always a safe bank, alphanumerical chars should work either, but what about others? Double quotes `"` are obviously an issue 😄.

Reply:
All the dashboard does is compare the secret from the token and the secret from the file, so almost anything should work.